src/components/send/components/entry_types/DateAndTime.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+import { Translate } from 'react-localize-redux'
+import StyledContainer from './Style.css'
+import getDateAndTime from '../../../../utils/getDateAndTime'
+
+const DateAndTime = ({ timeStamp, translate }) => {
+    return (
+        <StyledContainer>
+            <Translate id={translate} />
+            <div className='time'>{getDateAndTime(timeStamp)}</div>
```

Comment:
I propose we rename `getDateAndTime` `formatTimestampForLocale` (including the file name)